src/migrations/Version20241110195320.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20241110195320 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return '';
```

Comment:
You should ideally "name" all your migrations so you know what they contained once you get back to them after months